torch/export/dynamic_shapes.py

```diff
@@ -55,7 +55,7 @@ def Dim(name: str, *, min: Optional[int] = None, max: Optional[int] = None):
     """
     _min = 2 if min is None else builtins.max(min, 2)
     _max = sys.maxsize - 1 if max is None else builtins.min(max, sys.maxsize - 1)
-    assert _max > _min, f"Cannot create Dim with inconsistent min={min}, max={max}"
+    assert _max >= _min, f"Cannot create Dim with inconsistent min={min}, max={max}"
```

Comment:
Hmm, this might cause problems down the line because everywhere else in symbolic shapes we have special handling of "symbol is specialized to int" than encoding it as "symbol has same upper and lower bound." See below.